TextToSpeech/TextToSpeechJsonRpc.cpp

```diff
@@ -171,7 +176,11 @@ namespace Plugin {
         config.language = GET_STR(parameters, "language", "");
         config.voice = GET_STR(parameters, "voice", "");
         config.speechRate = GET_STR(parameters, "speechrate", "");
-        config.volume = (uint8_t) std::stod(GET_STR(parameters, "volume", "0.0"));
+        std::string proxyVolume = GET_STR(parameters, "volume", "0.0");
+
+        if(!InputValidation::Instance().validate("double_str", proxyVolume))
+            return Core::ERROR_GENERAL;
```

Comment:
When we bail out from SetConfiguration() prematurely, should we set TTS_Status value?